src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx

```diff
@@ -278,43 +312,39 @@ export const Deposit: React.FC<DepositProps> = ({
             wallet,
           })
 
-          const fees = await adapter.getFeeData({
+          const fees = (await estimateFees({
```

Comment:
we already are estimating the fees on `createBuildCustomTxInput`, we could extend it's return type to have access to `networkFeeCryptoBaseUnit` if we wanted (note: it is using average instead of fast which is honestly probably sufficient, but have noticed everything else is using fast in the savers)

Reply:
follow up